### DISCUSSION TITLE
Next.js Image component always loads image as 3840px

### DISCUSSION THREAD

**Discussion by 478732** [vercel/next.js#71445] [Help]
### Link to the code that reproduces this issue

https://codesandbox.io/p/devbox/7djqzn

```
import Image from "next/image";
import imageShapes from "../public/unsplash.jpg";

export default function Home() {
  return (
    <main>
      <div style={{ width: "24rem", height: "24rem" }}>
        <Image
          style={{ width: "100%", height: "100%", background: "black" }}
          src={imageShapes}
          alt=""
          sizes="1px"
        />
      </div>
    </main>
  );
}
```

### To Reproduce

1. Start the application in development mode
2. Visit the home page
3. Inspect the image
4. In the inspector, it's shown that the image is currently using the 3840px version as seen in the img "src" attribute.
6. This is the src of the img
7. src="/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Funsplash.549710ac.jpg&amp;w=3840&amp;q=75">
8. The "w=3840"  on the img src is being set automatically. I'm not sure how that value is computed.
<img width="1068" alt="ewew" src="https://github.com/user-attachments/assets/d781406f-91d4-4c77-9b26-cc514cb2e7de">

9. The code for this page is located in app/page.tsx

### Current vs. Expected behavior

I'm using the Next.js <Image> component to display an image, but I’m running into a problem where the image is always set to a width of 3840px. I have the "sizes" attribute set to 1px, but it doesn't seem to take effect.

Despite setting sizes="1px", the image URL generated has w=3840, which is much larger than expected. Here’s a screenshot showing the image using the 3840px version.

According to the Next.js documentation:https://nextjs.org/docs/pages/api-reference/components/image#width, the width and height for statically imported images don't need to be set.

Does anyone know why the image is defaulting to such a large size? How can I ensure it uses a smaller version based on the sizes attribute? Any help would be greatly appreciated!

### Provide environment information

```bash
google chrome
```

### Which area(s) are affected? (Select all that apply)

Image (next/image)

### Which stage(s) are affected? (Select all that apply)

next dev (local), next build (local), next start (local), Vercel (Deployed)

**Comment by samcx** [comment]
@478732 I updated some things (also updated `next` to the latest stable) and it works for me.

```
import Image from "next/image";

export default function Home() {
  return (
    <main>
        <Image
        src="/unsplash.jpg"
        alt="unsplash image"
        width={500}
        height={500}
        />
    </main>
  );
}
```

![CleanShot 2024-10-17 at 14 43 11@2x](https://github.com/user-attachments/assets/55399945-f680-401f-9e52-3c055180373e)

**Comment by madfcat** [comment]
I think you should dive deeper in the documentation to understand how the Image component in Next.js works.
Your image is not 3840px. It is only 16px.

<img width="1088" alt="image" src="https://github.com/user-attachments/assets/c74923c0-b8d2-47e6-9c0c-b8b81d80126a">



What you can do is remove you inline styles from div and use `sizes` attribute in a correct way like this:
```
import Image from "next/image";
import imageShapes from "../public/unsplash.jpg";

export default function Home() {
  return (
    <main>
      <div>
        <Image
          style={{ width: "100%", height: "100%" }}
          src={imageShapes}
          alt=""
          sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
        />
      </div>
    </main>
  );
}
```

Now, when you resize the window Image component will react to the width and server you with a suitable image size to save bandwidth.
<img width="798" alt="image" src="https://github.com/user-attachments/assets/08583bf3-c396-467e-8cf8-e9142c654a9e">
<img width="814" alt="image" src="https://github.com/user-attachments/assets/25f630ab-527a-4999-bb70-a6a66e27aef7">
<img width="813" alt="image" src="https://github.com/user-attachments/assets/ec410cd0-4041-431d-a205-47eb4fc3aa8c">

Playing with sizes attribute you can achieve the desired result.

Please, check the [official documentation](https://nextjs.org/docs/app/api-reference/components/image).
There is a really good [Next.js Image component YouTube tutorial](https://www.youtube.com/watch?v=ZKG8JBdgSos)
